Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM goreleaser/goreleaser:latest as builder
+
+ENV GORELEASER_CURRENT_TAG=latest
+
+WORKDIR /tmp/cirrus-ci-agent
+ADD . /tmp/cirrus-ci-agent/
+
+RUN goreleaser --snapshot
```

Comment:
I think `goreleaser build` here is more appropriate, no need to create archives and/or publish anything.